### PR TITLE
Added `copy.existing.pipeline` configuration.

### PR DESCRIPTION
**Pull request by rozza**
Allows indexes to be used during the copying process, use when there is any filtering done by the main pipeline

KAFKA-131

### CODE REVIEW

**Comment by rozza** [comment]
https://spruce.mongodb.com/version/5f50e1d63066154dae0e1a09/tasks

File: src/main/java/com/mongodb/kafka/connect/source/MongoSourceConfig.java [hunk 1]
```diff
@@ -207,6 +207,15 @@
       "The max size of the queue to use when copying data.";
   private static final int COPY_EXISTING_QUEUE_SIZE_DEFAULT = 16000;
 
+  public static final String COPY_EXISTING_PIPELINE_CONFIG = "copy.existing.pipeline";
+  private static final String COPY_EXISTING_PIPELINE_DISPLAY = "Copy existing initial pipeline";
+  private static final String COPY_EXISTING_PIPELINE_DOC =
+      "An inline JSON array with objects describing the pipeline operations to run when copying existing data.\n"
+          + "This can improve the use of indexes by the copying manager and making copying more efficient.\n"
```
Comment (jyemin) [inline src/main/java/com/mongodb/kafka/connect/source/MongoSourceConfig.java:214]: making => make

**Review by jyemin — APPROVED** [review]
One nit, but LGTM

**Comment by rozza** [comment]
Fixed typo, rebased and merged

**Pull request closed without merging**